src/Umbraco.Web.UI.Client/src/views/components/editor/umb-editor-header.html

```diff
@@ -37,7 +37,7 @@
                                umb-auto-focus
                                val-server-field="Name"
                                required
-                               autocomplete="off" />
+                               autocomplete="off" maxlength="255"/>
```

Comment:
Just a little suggestion to align the new attribute on a new line:
```suggestion
                               autocomplete="off"
                               maxlength="255"/>
```